interfaces.go

```diff
@@ -143,8 +146,42 @@ type Exception struct {
 
 type EventID string
 
-// https://docs.sentry.io/development/sdk-dev/event-payloads/
+// TraceContext describes the context of the trace.
+type TraceContext struct {
+	TraceID     string `json:"trace_id"`
+	SpanID      string `json:"span_id"`
+	Op          string `json:"op,omitempty"`
+	Description string `json:"description,omitempty"`
+}
+
+// Span describes a AM Span following the Sentry format.
```

Comment:
Might be hard to follow in API docs what is "AM".
We could start with something simpler. Sentry is redundant here (there is no confusion with OTel).

```suggestion
// Span describes a timed part of a trace.
```

or 

```suggestion
// Span describes a span.
```

or open for other possibilities ;)

Reply:
Replaced with `Span describes a timed unit of work in a trace`